dune/session.py

```diff
@@ -16,6 +16,10 @@ class HostAction:
     def get_arg_spec(cls, faction=None, game_state=None):
         return Args()
 
+    @classmethod
+    def get_is_blocking(cls):
```

Comment:
I couldn't use the host without adding this.

Reply:
ah, word... if doesn't quack..